lib/rex/powershell/command.rb

```diff
@@ -340,9 +363,9 @@ def self.cmd_psh_payload(pay, payload_arch, template_path, opts = {})
       command_args[:command] = final_payload
     end
 
-    psh_command = generate_psh_command_line(command_args)
+    psh_command = opts[:exec_in_place] ? "#{command_args[:command]}" : generate_psh_command_line(command_args)
```

Comment:
could you please refactor this so it is not using a ternary operator? ternary operators are the devil